src/test/java/com/easypost/Fixture.java

```diff
@@ -0,0 +1,281 @@
+     */
+    public static Map<String, Object> basicCustomsItem() {
+        Map<String, Object> customsItem = new HashMap<>();
+
+        customsItem.put("description", "Sweet shirts");
+        customsItem.put("quantity", "2");
+        customsItem.put("weight", "11");
+        customsItem.put("value", "23");
+        customsItem.put("hs_tariff_number", "654321");
+        customsItem.put("origin_country", "US");
+
+        return customsItem;
+    }
+
+    /**
+     * Basic customs Info that is reusable in all test coverage.
+     *
+     * @return A map that have the customs Info info filled.
+     */
+    public static Map<String, Object> basicCustomsInfo() {
+        Map<String, Object> customsInfo = new HashMap<>();
+
+        customsInfo.put("eel_pfc", "NOEEI 30.37(a)");
+        customsInfo.put("customs_certify", true);
+        customsInfo.put("customs_signer", "Steve Brule");
+        customsInfo.put("contents_type", "merchandise");
+        customsInfo.put("contents_explanation", "");
+        customsInfo.put("restriction_type", "none");
+        customsInfo.put("non_delivery_option", "return");
+        customsInfo.put("custom_items", basicCustomsItem());
+
+        return customsInfo;
+    }
+
+    /**
+     * Basic tax identifier that is reusable in all test coverage.
+     *
+     * @return A map that have the tax identifier info filled.
+     */
+    public static Map<String, Object> taxIdentifier() {
+        Map<String, Object> taxIdentifier = new HashMap<>();
+
+        taxIdentifier.put("entity", "SENDER");
+        taxIdentifier.put("tax_id_type", "IOSS");
+        taxIdentifier.put("tax_id", "12345");
+        taxIdentifier.put("issuing_country", "GB");
+
+        return taxIdentifier;
+    }
+
+    /**
+     * Basic shipment that is reusable in all test coverage.
+     *
+     * @return A map that have the shipment info filled.
+     */
+    public static Map<String, Object> basicShipment() {
+        Map<String, Object> basicShipment = new HashMap<>();
+
+        basicShipment.put("to_address", basicAddress());
+        basicShipment.put("from_address", basicAddress());
+        basicShipment.put("parcel", basicParcel());
+
+        return basicShipment;
+    }
+
+    /**
+     * Shipment that has all detail and is reusable in all test coverage.
+     *
+     * @return A map that have the shipment info filled in detail.
+     */
+    public static Map<String, Object> fullShipment() {
+        Map<String, Object> fullShipment = new HashMap<>();
+        Map<String, Object> options = new HashMap<>();
+
+        options.put("label_format", "PNG");
+        options.put("invoice_number", "123");
+        fullShipment.put("to_address", basicAddress());
+        fullShipment.put("from_address", basicAddress());
+        fullShipment.put("parcel", basicParcel());
+        fullShipment.put("customs_info", basicCustomsInfo());
+        fullShipment.put("reference", "123");
```

Comment:
What happens when you make this field an int? It probably complains because it's strongly typed eh?

Reply:
The `reference` attribute in shipment is String, so yes it's strongly typed.

Reply:
I ask because PHP for instance will lazily let an integer through and convert it on the way to string before sending it to the API; however, Ruby won't. I had to adjust them to strings in Ruby. The reason they are ints in the PHP lib is to test our serialization, that's why those comments were there. As this is a strongly typed language, let's keep these as they are.